docs/glossary.md

```diff
@@ -30,4 +30,4 @@
 + **Productions server**: A server on AppSpot. Can be the live server or a staging server.
 + **Spec**: System Specification.
 + **Staging green**: All test are passing against you own deployment on the GAE.
-+ **Staging server**: The server instance you created on AppSpot.
\ No newline at end of file
++ **Staging server**: The server instance you created on AppSpot.
```

Comment:
@wkurniawan07  Actually whenever I edit the repo url and commit that file. It adds extra line at the end of file, I try to clear that line and again committed that file, but still commit shows no changes in file. I tried to edit this file in my various editors in my PC as well as in github editor, issue remain same. Do you have any idea how to prevent it.